client/.eslintrc.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    commonjs: true
+  },
+  globals: {
+    // Webpack plugin adds `process` global
+    process: false
```

Comment:
Badly worded comment from me, I meant exactly that it defines the existence of a global for ESLint to know.

Based on the comment in the code, it looked like the process global was attempted to be removed from the global scope. But apparently you knew exactly what you were doing and I made an incorrect interpretation. :)